internal/api/approve_freight_v1alpha1_test.go

```diff
@@ -345,20 +360,27 @@ func TestApproveFreight(t *testing.T) {
 			},
 			assertions: func(
 				t *testing.T,
+				er *fakekubeclient.EventRecorder,
```

Comment:
Nit: `er` looks too close to the symbol commonly used for errors -- `err`. I suspect this will be the source of confusion now and then unless one reading the code brings extra scrutiny to bear on what they're reading. I would name this differently, but like I said... this is a nit. Dealer's choice.

Reply:
I historically have ended up with using either `events` or `recorder` in these kind of situations.

Reply:
Changed to use `recorder`: https://github.com/akuity/kargo/pull/1739/commits/dc92e6044fb6b7216728c29a255c14fe327ce0a3